website/docs/reference/project-configs/asset-paths.md

```diff
@@ -21,7 +21,7 @@ Optionally specify a custom list of directories to copy to the `target` director
 </Changelog>
 
 ## Default
-By default, dbt will not compile any additional files as part of `docs generate`, i.e. `analysis-paths: []`
+By default, dbt will not compile any additional files as part of `docs generate`, i.e. `asset-paths: []`
```

Comment:
thanks for opening this up @kopackiw! i believe what the docs is saying is that your analyses (saved in `analysis-path`) will not be generated by a `dbt docs generate`. I've suggested the below to clarify a bit further:

```suggestion
By default, dbt will not compile additional files as part of `docs generate`. For example &mdash;  your analyses, saved in `analysis-paths: []`, won't be generated by `dbt docs generate`. 
```

Reply:
I think that the issue that is being reported is that the `Default` section of the `asset-paths` page talks about `analysis-paths`.

I'd change it to: 
By default, dbt will not copy any additional files as part of `docs generate`, i.e. `asset-paths: []`

Replacing `copy` by `compile` and using the correct config item.